mcts/src/main/java/org/motechproject/nms/mcts/service/impl/MctsWebServiceFacadeImpl.java

```diff
@@ -51,7 +51,9 @@ public ChildrenDataSet getChildrenData(LocalDate from, LocalDate to, URL endpoin
 
         try {
             validChildrenDataResponse(result, stateId);
-            return (ChildrenDataSet) MarshallUtils.unmarshall(result.get_any()[1].getChildren().get(0).toString(), ChildrenDataSet.class);
+            return (result.get_any()[1].getChildren() == null) ?
```

Comment:
Wouldn't extracting `result.get_any()[1].getChildren()` into a variable here and in the occurrences below make the code more readable?